modules/phase_field/src/kernels/ACBulk.C

```diff
@@ -10,44 +10,59 @@ template<>
 InputParameters validParams<ACBulk>()
 {
   InputParameters params = validParams<KernelValue>();
-  params.addClassDescription("Allen-Cahn Kernel");
+  params.addClassDescription("Allen-Cahn bulk energy kernel");
   params.addParam<MaterialPropertyName>("mob_name", "L", "The mobility used with the kernel");
+  params.addCoupledVar("args", "Vector of additional arguments");
```

Comment:
I was thinking along the same lines, but then the "args" name doesn't make sense. We currently use "args" for this purpose in various locations so if we change it here, we should change it everywhere.